### PR TITLE
Reworked big module for using Work Items

### PR DESCRIPTION
**Pull request by mishaone**
* Changed a lot of passive voice to active voice
* Rewrote things for consistency, decided to use conventions like marking up views as menus rather than with asterisks, etc.
* Removed unnecessary information, reworked rest for hopefully improved flow, etc.

### CODE REVIEW

File: user_guide/content/modules/working_with_an_existing_work_item.adoc [hunk 1]
```diff
@@ -1,90 +1,68 @@
...
 = Working with an Existing Work Item
-//redo this, stopped here
-Use the Planner in the *Plan* tab to track your Work Items and Iterations using either the list view or the board view. The *Plan* tab is split into two panes:
 
-* The left pane displays the *Backlog* and *Board* views of Iterations and Work Items. For more information see <<about_iterations,About Iterations>>.
-* The right pane displays the Work Items within the selected Iteration or the backlog.
+Click btn:[Plan] at the top of the page to view the Planner.
 
-You can plan and track your Work Items in the right pane using either the hierarchical or flat list view of Work Items on the menu:Plan[Backlog] subtab or the Kanban-based board view on the menu:Plan[Board] subtab.
+The *Backlog* view is the default view in the Planner and it displays all the Iterations included in the space. Iterations are sorted into one of the following categories:
```
Comment (Preeticp) [inline user_guide/content/modules/working_with_an_existing_work_item.adoc]: The information on line 9 is the crucial difference between the list view and board view. The Plan tab is split into two Panes,(line 4) one talks about Iterations the other about Work Items. 
Work Items are displayed in two ways, List and Kanban style. In the Backlog view, Work items are displayed in a list view, this part should be mentioned in the introductory paragraph, imho. User has the choice to work with Work items either using the list view or the Board view based on his choice.
Currently, in the Backlog section, Iterations are being described (line 8) which is mentioned in detail in the Iterations topic. We can simply link to the section as in line 6. Can we discuss this please?

File: user_guide/content/modules/working_with_an_existing_work_item.adoc [hunk 2]
```diff
@@ -1,90 +1,68 @@
...
-* Owner
-* Area
-* Iteration
-* Description
-* Relations with other Work Items
-* Comments
-
-+
-See <<using_the_detailed_method>> for more information on modifying the above attributes.
+Alternatively, click the options (image:kabob.png[title="More"]) icon for the relevant Work Item and select *Open* from the displayed options. Make the required changes to the Work Item details and then save with the check button.
```
Comment (Preeticp) [inline user_guide/content/modules/working_with_an_existing_work_item.adoc]: Looks good, +1.

File: user_guide/content/modules/working_with_an_existing_work_item.adoc (hunk 1 above)
Comment (mishaone) [inline user_guide/content/modules/working_with_an_existing_work_item.adoc]: Happy to have a discussion. Shall we wait for the meeting so we all get to be involved? :)

Comment (Preeticp) [inline user_guide/content/modules/working_with_an_existing_work_item.adoc]: Sounds good :)

**Comment by fabric8cd** [comment]
@mishaone snapshot fabric8/fabric8-online-docs:SNAPSHOT-PR-1-1 is available for testing at http://fabric8-online-docs-app-pr-1-fabric8-online-docs.badger.fabric8.io

**Comment by mishaone** [comment]
Closing since the UI is changing anyway. We will review this in the future once the major changes are done.

**Pull request closed without merging**